wled00/data/index.js

```
@@ -2820,56 +2804,53 @@ function rSegs()
-		} catch (e) {}
-	}
+function loadPalettesData() {
+	return new Promise((resolve) => {
+		if (palettesData) return resolve(); // already loaded
+		const lsKey = "wledPalx";
+		var lsPalData = localStorage.getItem(lsKey);
+		if (lsPalData) {
+			try {
+				var d = JSON.parse(lsPalData);
+				if (d && d.vid == d.vid) {
+					palettesData = d.p;
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)